src/commands/mock.ts

```diff
@@ -47,5 +47,31 @@ export const Mock = {
         }
         mockTargets.add(target.id);
         await interaction.reply(`Sure thing, I will begin mocking ${target}!`);
+    },
+
+    Effect: async (message: Message): Promise<void> => {
```

Comment:
I would include some documentation on what this function does.

Also, `Effect` should be `effect`. Function names are usually in camelcase.